olm-catalog/serverless-operator/project.yaml

```diff
@@ -1,78 +1,78 @@
 project:
-  name: serverless-operator
-  # When bumping the Operator to a new version (major and minor), make sure to also update
-  # all components in `dependencies.previous` to the same versions as `dependencies` in the same PR.
-  # Otherwise, the upgrade tests will not pass, as we have a different SO version with the same bundle contents.
-  # Also make sure to update values under `olm.previous` by copying from `olm.replaces` and `olm.skipRange`.
-  version: 1.35.0
+    name: serverless-operator
+    # When bumping the Operator to a new version (major and minor), make sure to also update
```

Comment:
Nit. Not sure the formatting change is intentional?

Reply:
the command `go run ./hack/cmd/bumpso/bumpso.go --branch release-1.35` decided the format for some reason